packages/circuit-ui/components/RadioButton/RadioButton.tsx

```diff
@@ -149,7 +154,9 @@ const inputBaseStyles = css`
   }
 
   &:disabled + label,
-  &[disabled] + label {
+  &:disabled + label span,
+  &[disabled] + label,
+  &[disabled] + label span {
```

Comment:
This selector change isn't needed anymore since the FieldDescription component defines its own disabled styles.